extensions/schema/src/main/java/com/microsoft/dagx/schema/policy/PolicySchema.java

```diff
@@ -12,6 +12,7 @@ protected void addAttributes() {
         attributes.add(new SchemaAttribute("serialized", true));
         attributes.add(new SchemaAttribute("validity-start", false));
         attributes.add(new SchemaAttribute("validity-end", false));
+        attributes.add(new SchemaAttribute("policyId", false));
```

Comment:
Isn't the policy schema used in another way than the other schemas? Maybe my understanding of the schemas is wrong. As far as I understand schema is something like a generic data schema which is used by the provisioning of data and during the transfer of data.
From my understanding a policy would not fall into this category, because its not provisioned and not transfered.